server/src/main/java/org/elasticsearch/index/mapper/FieldMapper.java

```diff
@@ -400,17 +402,6 @@ private void mergeSharedOptions(FieldMapper mergeWith, List<String> conflicts) {
         if (fieldType.storeTermVectorPayloads() != other.storeTermVectorPayloads()) {
             conflicts.add("mapper [" + name() + "] has different [store_term_vector_payloads] values");
         }
-
```

Comment:
Nothing calls this anymore (all mappers that use analyzers have been parametrized) so it is safe to remove.